docs/docs/drivers/microsoft-sqlserver.md

```
@@ -165,4 +165,61 @@ Based on [tedious](https://tediousjs.github.io/node-mssql/) MSSQL implementation
+`int`, `bigint`, `bit`, `decimal`, `money`, `numeric`, `smallint`, `smallmoney`, `tinyint`, `float`, `real`, `date`, `datetime2`, `datetime`, `datetimeoffset`, `smalldatetime`, `time`, `char`, `varchar`, `text`, `nchar`, `nvarchar`, `ntext`, `binary`, `image`, `varbinary`, `hierarchyid`, `sql_variant`, `timestamp`, `uniqueidentifier`, `xml`, `geometry`, `geography`, `rowversion`, `vector`
+
+### Vector Type (vector)
+
+The `vector` data type is available in SQL Server for storing high-dimensional vectors, commonly used for:
+
+-   Semantic search with embeddings
+-   Recommendation systems
+-   Similarity matching
+-   Machine learning applications
+
+NOTE: general `halfvec` type support is unavailable because this feature is still in preview: https://learn.microsoft.com/en-us/sql/t-sql/data-types/vector-data-type?view=sql-server-ver17&tabs=csharp
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)